src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheck.java

```diff
@@ -61,6 +61,58 @@
  * the allowInSwitchCase property to true and include all statements
  * of the case in the block.
  * </p>
+ * <ul>
+ * <li>
+ * Property {@code allowInSwitchCase} - Allow nested blocks if they are the
+ * only child of a switch case.
+ * Type is {@code boolean}.
+ * Default value is {@code false}.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="AvoidNestedBlocks"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() {
+ *   int myInteger = 0;
+ *   {
+ *     myInteger = 2;    //NOT OK, violation : Avoid nested blocks.
```

Comment:
`//NOT OK, violation : Avoid nested blocks.` just `// violation` is enough.

Reply:
Done